src/main/scala/li/cil/oc/integration/ic2/EventHandlerIndustrialCraft2.scala

```diff
@@ -1,26 +1,54 @@
 package li.cil.oc.integration.ic2
 
 import cpw.mods.fml.common.eventhandler.SubscribeEvent
+import ic2.api.crops.ICropTile
 import ic2.api.item.ElectricItem
 import ic2.api.item.IElectricItem
 import ic2.api.item.ISpecialElectricItem
 import ic2.core.item.tool.ItemToolWrench
-import li.cil.oc.api.event.RobotUsedToolEvent
+import li.cil.oc.api.event.{GeolyzerEvent, RobotUsedToolEvent}
 import li.cil.oc.integration.util.Power
 import net.minecraft.entity.player.EntityPlayer
 import net.minecraft.item.ItemStack
+import scala.collection.convert.WrapAsScala._
 
 object EventHandlerIndustrialCraft2 {
+  @SubscribeEvent
+  def onGeolyzerAnalyze(e: GeolyzerEvent.Analyze) {
+    val world = e.host.world
+    world.getTileEntity(e.x, e.y, e.z) match {
+      case crop: ICropTile =>
+        crop.setScanLevel(4)
+        val cc = crop.getCrop
+        if (cc != null) {
+          e.data += "crop:name" -> cc.name()
```

Comment:
AgriCraft doesn't use a `crop:` prefix. As per the fluid container discussion, would this be better without the prefix, or in a separate "crop" map?